JAICore/jaicore-ml/src/main/java/jaicore/ml/core/evaluation/measure/multilabel/OverviewMeasureLoss.java

```diff
@@ -0,0 +1,22 @@
+package jaicore.ml.core.evaluation.measure.multilabel;
+
+/**
+* Measure combining exact match, hamming loss, f1macroavgL and rankloss. Here
+* implemented in inverse.
+*
+* @author helegraf, mwever
+*
+*/
+public class OverviewMeasureLoss extends ADecomposableMultilabelMeasure {

```

Comment:
This is really a very strange measure (I believe). Even if you keep this, a name different from "Overview" would be a good thing. This is somehow a cross-metric "average" measure. I do not believe that this number has any reasonable semantic.